scripts/qa_crowbarsetup.sh

```diff
@@ -3581,7 +3581,7 @@ function oncontroller_magnum_service_setup
             --property os_distro=fedora-atomic magnum-service-image
     fi
 
-    if ! openstack image list --f value -c Name | grep -q "^SLE12SP1-JeOS-k8s-magnum$"; then
+    if ! openstack image list --f value -c Name | grep -q "^sles-openstack-magnum-kubernetes$"; then
```

Comment:
maybe you want to use $service_image_name here and just append the .qcow2 in the url line above instead (this needs to have a arch suffix anyway)